File: app/src/main/java/com/chooloo/www/callmanager/ui/about/AboutMvpPresenter.java
package com.chooloo.www.callmanager.ui.about;

import com.chooloo.www.callmanager.ui.base.MvpPresenter;

public interface AboutMvpPresenter<V extends AboutMvpView> extends MvpPresenter<V> {
    void onChangelogClick();

    void onOpenSourceClick();

    void onSendEmailClick();

    void onReportBugClick();

    void onRateAppClick();

    void onDonateClick();
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/about/AboutMvpPresenter.kt
package com.chooloo.www.callmanager.ui.about

import com.chooloo.www.callmanager.ui.base.MvpPresenter

interface AboutMvpPresenter<V : AboutMvpView?> : MvpPresenter<V> {
    fun onChangelogClick()
    fun onOpenSourceClick()
    fun onSendEmailClick()
    fun onReportBugClick()
    fun onRateAppClick()
    fun onDonateClick()
}